libcnb-test/src/build_config.rs

```diff
@@ -251,6 +252,8 @@ impl BuildConfig {
 pub enum BuildpackReference {
```

Comment:
The new feature (plus enum variant) needs a changelog entry

Reply:
Added